lib/elixir/test/elixir/function_test.exs

```diff
@@ -0,0 +1,80 @@
+Code.require_file("test_helper.exs", __DIR__)
+
+defmodule DummyFunction do
+  def function_with_arity_0 do
+    true
+  end
+
+  def zero?(0), do: true
+  def zero?(_), do: false
+end
+
+defmodule FunctionTest do
+  use ExUnit.Case, async: true
+
+  doctest Function
+  import Function
+
+  @information_keys_for_named [:type, :module, :arity, :name, :env]
+  @information_keys_for_anonymous @information_keys_for_named ++
+                                    [:pid, :index, :new_index, :new_uniq, :uniq]
+
+  describe "capture/3" do
+    test "that it can capture module functions with arity 0" do
```

Comment:
We can remove the "that it can" and "it" prefixes from tests. This should read as: `test "captures module functions with arity 0"`. Same for the tests below.